src/app/core/services/connector-limits.service.ts

```diff
@@ -0,0 +1,49 @@
+import {Injectable} from '@angular/core';
+import {Observable, of} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {ConnectorLimits} from '@sovity.de/edc-client';
+import {ActiveFeatureSet} from '../config/active-feature-set';
+import {EdcApiService} from './api/edc-api.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ConnectorLimitsService {
+  constructor(
+    private edcApiService: EdcApiService,
+    private activeFeatureSet: ActiveFeatureSet,
+  ) {}
+
+  fetchLimits(): Observable<ConnectorLimits> {
+    return this.edcApiService.getEnterpriseEditionConnectorLimits();
+  }
```

Comment:
this can be inlined, all calls to this service should use the same simplified function call, e.g. "limitsExceeded"

or: it can be private and should  be moved to the bottom as it is less important. the main method you'd want people to use you should put at the top